manifests/b/BandicamCompany/Bandicam/3.2.2.1111/BandicamCompany.Bandicam.locale.en-US.yaml

```diff
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=QUSU.5-1-19041-1237
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: BandicamCompany.Bandicam
+PackageVersion: 3.2.2.1111
+PackageLocale: en-US
+Publisher: Bandicam.com
```

Comment:
```suggestion
Publisher: Bandisoft.com
```

Reply:
Thank you @ItzLevvie; I didn't catch that it had changed halfway through